using_admittance_controller.md

```diff
@@ -0,0 +1,84 @@
+# Using admittance controller
```

Comment:
I think this file should go in the /admittance_demo directory, not the root directory. Same for `admittance_controller.repos`